.github/workflows/go.yml

```diff
@@ -30,5 +34,20 @@ jobs:
             dep ensure
         fi
 
-    - name: Build
-      run: go build -v .
+    - name: Build for linux/amd64
+      env:
+        GO_VERSION: v1.14
+        TIME: "${{ steps.current-time.outputs.time }}"
+      run: GOOS=linux GOARCH=amd64 go build -v -o bin/kadok -ldflags "-X main.Version=$GITHUB_REF -X main.BuildDate=$TIME -X main.GitCommit=$GITHUB_SHA -X main.GoVersion=$GO_VERSION"
+
+    - name: Build for windows/amd64
+      env:
+        GO_VERSION: v1.14
+        TIME: "${{ steps.current-time.outputs.time }}"
+      run: GOOS=windows GOARCH=amd64 go build -v -o bin/kadok.exe -ldflags "-X main.Version=$GITHUB_REF -X main.BuildDate=$TIME -X main.GitCommit=$GITHUB_SHA -X main.GoVersion=$GO_VERSION"
+
+    - name: Upload Build Artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: kadok
+        path: bin/*
```

Comment:
missing new line

it may be interesting to add an editorconfig file to enforce the new line behavior

Reply:
Issue #35 created